src/main/java/com/google/devtools/build/lib/bazel/rules/BazelStrategyModule.java

```diff
@@ -90,5 +91,9 @@ public void registerSpawnStrategies(
     for (Map.Entry<RegexFilter, List<String>> entry : options.strategyByRegexp) {
       registryBuilder.addDescriptionFilter(entry.getKey(), entry.getValue());
     }
+
+    for (Map.Entry<String, List<String>> strategy : options.allowedStrategiesByExecPlatform) {
+      registryBuilder.addExecPlatformFilter(Label.parseCanonicalUnchecked(strategy.getKey()), strategy.getValue());
```

Comment:
We should comvert these to a Label in the Options class, if possible, to ensure that the conversion context is populated.

Consider a case where `bazel` is run in a subdirectory, we still want relative labels to work sensibly with flags.